zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -279,6 +280,112 @@ def channels_to_zerver_stream(slack_dir: str, realm_id: int, added_users: AddedU
     print('######### IMPORTING STREAMS FINISHED #########\n')
     return zerver_defaultstream, zerver_stream, added_channels, zerver_subscription, zerver_recipient
 
+def channel_message_to_zerver_message(constants: List[Any], channel: str,
+                                      added_users: AddedUsersT, added_channels: AddedChannelsT,
+                                      zerver_userprofile: List[ZerverFieldsT],
+                                      zerver_subscription: List[ZerverFieldsT],
+                                      ids: List[int]) -> Tuple[List[ZerverFieldsT],
+                                                               List[ZerverFieldsT]]:
+    """
+    Returns:
+    1. zerver_message, which is a list of the messages
+    2. zerver_usermessage, which is a list of the usermessages
+    """
+    slack_dir, REALM_ID = constants
+    message_id, usermessage_id = ids
+    json_names = os.listdir(slack_dir + '/' + channel)
+    users = json.load(open(slack_dir + '/users.json'))
+    zerver_message = []
+    zerver_usermessage = []
+
+    # Markdown mapping
+    def convert_to_zulip_markdown(text: str) -> Tuple[str, List[int], bool]:
```

Comment:
Should there be a TODO for mapping `_stuff_` to `*stuff*` ?

Reply:
@rht I have added the regex. Can you go through this once?